build/scripts/build.ps1

```diff
@@ -250,7 +251,7 @@ function Build-Artifacts() {
         Run-MSBuild "Compilers.sln" -useDotnetBuild
     }
     elseif ($build) {
-        Run-MSBuild "Roslyn.sln" "/p:DeployExtension=$deployExtensions"
+        Run-MSBuild "Roslyn.sln" $(if (-not $deployExtensions) {"/p:DeployExtension=false"})
```

Comment:
Was not aware Powershell allowed for such complex interpolation. Did you find docs expressing the full limits of it?

Reply:
I found this: https://stackoverflow.com/a/34311158/138304

Then I tried simplifying it by omitting `else {""}` and found that it worked, so I left it.